resources/views/home.blade.php

```diff
@@ -80,7 +80,7 @@
                           <h2 class="mb-4 h5">{{ __('Servers') }}</h2>
                       </div>
                       <div>
-                          <button class="btn btn-primary"><i class="fas fa-plus me-2"></i> {{__('Create server')}}</button>
+                          <a href="{{route("checkout.store")}}"><button class="btn btn-primary"><i class="fas fa-plus me-2"></i> {{__('Create server')}}</button></a>
```

Comment:
you do not put an <a> tag around a button, you simply change the <button> to an a tag and keep the classes on it